portx/modules/ROOT/pages/_partials/documents-view.adoc

```diff
@@ -1,51 +1,54 @@
 == Documents View
```

Comment:
Commenting on images here.

Errors View - Don't think we can use WSO2, but let's ask Brad